iroh-cli/src/config.rs

```diff
@@ -19,22 +17,16 @@ use iroh::node::GcPolicy;
 use iroh::sync::{AuthorId, NamespaceId};
 use parking_lot::RwLock;
 use serde::{Deserialize, Serialize};
-use tracing::debug;
-
-/// CONFIG_FILE_NAME is the name of the optional config file located in the iroh home directory
-pub(crate) const CONFIG_FILE_NAME: &str = "iroh.config.toml";
-
-/// ENV_PREFIX should be used along side the config field name to set a config field using
-/// environment variables
-/// For example, `IROH_PATH=/path/to/config` would set the value of the `Config.path` field
-pub(crate) const ENV_PREFIX: &str = "IROH";
 
 const ENV_AUTHOR: &str = "AUTHOR";
 const ENV_DOC: &str = "DOC";
 
+/// CONFIG_FILE_NAME is the name of the optional config file located in the iroh home directory
+pub(crate) const CONFIG_FILE_NAME: &str = "iroh.config.toml";
+
 /// Fetches the environment variable `IROH_<key>` from the current process.
 pub(crate) fn env_var(key: &str) -> std::result::Result<String, env::VarError> {
-    env::var(format!("{ENV_PREFIX}_{key}"))
+    env::var(format!("IROH_{key}"))
```

Comment:
fixed

Reply:
or at least simplified

Reply:
for extra bonus points name the consts `ENV_IROH_CONFIG_DIR` etc, so that they contain the same greppable string as the env var itself.  but as it is now it's already great.  thanks!